src/Uno.UI/UI/Xaml/ApplicationActivity.Android.cs

```diff
@@ -61,6 +61,9 @@ private void OnInputPaneVisibilityChanged(InputPane sender, InputPaneVisibilityE
 				// using either SoftInput.AdjustResize or SoftInput.AdjustPan.
 				args.EnsuredFocusedElementInView = true;
 			}
+
+			// This handle when the soft bar is showed / hidden on some devices and recalculate bounds
+			RaiseConfigurationChanges();
```

Comment:
This is an event handler for the keyboard showing or hiding. If it's being called when the bottom bar shows/hides, then there's a bug somewhere.

Reply:
This is raised because of ViewTreeObserver.IOnGlobalLayoutListener.OnGlobalLayout from KeyboardRectProvider.Android which measures the difference between the screen size and the popupView.
It actually triggers on some devices (such as LG G7 ThinQ) because the user can choose to hide or show the bottom soft bar.
I will try to fix that.

Reply:
I can see why OnGlobalLayout might be raised, but it seems like a bug if InputPane.OccludedRect is also changing.

Reply:
Yes, but it's beacause of the keyboardRect calculated in OnGlobalLayout. The value changes when the soft bar is showed / hidden and then it changes the InputPane.OccludedRect value.

Reply:
Should be fixed :)